articles/vpn-gateway/azure-vpn-client-optional-configurations.md

```diff
@@ -153,15 +160,18 @@ You block (exclude) routes. Modify the downloaded profile XML file and add the *
 		<route>
 			<destination>x.x.x.x</destination><mask>24</mask>
 		</route>
+		<route>
+    			<destination>y.y.y.y</destination><mask>24</mask>
+    		</route>
 	</excluderoutes>
     
 </clientconfig>
 </azvpnprofile>
 ```
 
 > [!NOTE]
-> - The default status for clientconfig tag is <clientconfig i:nil="true" />, which can be modified based on the requirement.
-> - Duplicate clientconfig tag is not supported on macOS, so make sure the clientconfig tag is not duplicated in the XML file.
+> - To include/exclude multiple destination routes, put each destination address under a separate route tag _(as shown in the above exmaples)_, as multiple destination address in single route tag will not work.
+> - In case you encounter the error "_**Destination cannot be empty or have more than one entry inside route tag**_", check the profile xml file and ensure that includeroutes/excluderoutes section has only one destination address inside a route tag.
```

Comment:
```suggestion
> - To include/exclude multiple destination routes, put each destination address under a separate route tag _(as shown in the above examples)_, because multiple destination addresses in a single route tag won't work.
> - If you encounter the error "_Destination cannot be empty or have more than one entry inside route tag_", check the profile XML file and ensure that the includeroutes/excluderoutes section has only one destination address inside a route tag.
```
Edits for Microsoft style